lib/index.js

```diff
@@ -0,0 +1,6 @@
+let { Given, When, Then } = require('@cucumber/cucumber');
```

Comment:
`const`?

Reply:
essentially the same `const` vs `let` remark that I already left in the `wdi5-fe-selectors` repo

Reply:
done